### PR TITLE
[chore] basic app-setup

### PR DESCRIPTION
**Pull request by punk1234**
`PR` by @Lanrey

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,145 @@
+{
+    "name": "funcbackend",
```
Comment (punk1234) [inline package.json]:
```suggestion
    "name": "func-flow-backend",
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    "type": "module",
+    "version": "0.0.1",
+    "description": "Backend for Func Fun Backend",
```
Comment (punk1234) [inline package.json]: `Backend for Func Fun Backend` ??

File: package.json [hunk 3]
```diff
@@ -0,0 +1,145 @@
+{
```
Comment (punk1234) [inline package.json:1]: @Lanrey , kindly clean-up file such that we only have dependencies that are being used for now.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+      "rimraf": "^3.0.2",
+      "sequelize": "^6.28.0",
+      "sequelize-typescript": "^2.1.5",
+      "supertest": "^6.3.3",
+      "ts-jest": "^29.0.3",
+      "typedi": "^0.10.0",
+      "until-promise": "^0.3.0",
+      "uuid": "^9.0.0"
+    },
+    "scripts": {
```
Comment (punk1234) [inline package.json]: Kindly clean-up scripts as we will be using `mongodb` database

File: src/config/constants.ts [hunk 5]
```diff
@@ -0,0 +1,56 @@
+import path, { join } from "path";
+import dotenv from "dotenv";
+import { expand } from "dotenv-expand";
+import findRoot from "find-root";
+import { boolean } from "boolean";
```
Comment (punk1234) [inline src/config/constants.ts]: I think this is not being used here

File: .gitignore [hunk 6]
```diff
@@ -0,0 +1,105 @@
+# Logs
```
Comment (punk1234) [inline .gitignore:1]: [OPTIONAL COMMENT]: Can clean this up to have only what we need to ignore

File: package.json [hunk 7]
```diff
@@ -0,0 +1,145 @@
+{
+    "name": "funcbackend",
+    "type": "module",
+    "version": "0.0.1",
+    "description": "Backend for Func Fun Backend",
+    "main": "src/app.ts",
+    "author": "Exeque",
```
Comment (punk1234) [inline package.json]: Lol, please use `<FUNC-FLOW TEAM>`

**Comment by Lanrey** [comment]
@punk1234 PR reviewed and fixed

**Review by punk1234 — COMMENTED** [review]
@Lanrey , you pushed `package.json` changes to `dev`, kindly revert. All changes to `dev` must go through review in a `PR`.

File: package.json [hunk 8]
```diff
@@ -80,15 +83,25 @@
       "jest-mock-kafkajs": "github:Wei-Zou/jest-mock-kafkajs",
       "joi": "^17.6.2",
       "jsonwebtoken": "^8.5.1",
+      "lodash": "^4.17.21",
+      "lodash-es": "^4.17.21",
+      "mathjs": "^11.5.0",
```
Comment (punk1234) [inline package.json]: @Lanrey , there are still some `npm packages` we are not using like `mathjs`, `event-dispatch` & others

File: package.json [hunk 9]
```diff
@@ -80,15 +83,25 @@
...
+      "moment": "^2.29.4",
+      "morgan": "^1.10.0",
+      "multer": "^1.4.5-lts.1",
+      "node-geocoder": "^4.2.0",
+      "nodemailer": "^6.8.0",
       "moment": "^2.29.4",
       "morgan": "^1.10.0",
       "multer": "^1.4.5-lts.1",
```
Comment (punk1234) [inline package.json]: @Lanrey , we are not using `multer` yet, kindly remove.

**Review by Lanrey — COMMENTED** [review]
@punk1234 Cleaned this up already. Thanks

**Comment by Lanrey** [comment]
@punk1234 I have cleaned this up

**Review by Lanrey — APPROVED** [review]

**Pull request merged**